org.oasis.xdita/dtd/lw-map.mod

```diff
@@ -82,7 +81,7 @@ PUBLIC "-//OASIS//ELEMENTS LIGHTWEIGHT DITA Map//EN"
            
              
 <!--                    LONG NAME: Key Definition  -->
-<!ELEMENT keydef	(topicmeta?, data*)        >
+<!ELEMENT keydef	(topicmeta)        >

```

Comment:
This turns topicmeta from an optional child to required, was that intentional?

Reply:
Thanks for catching it, Robert. No. it continues to be optional.